apps/web-giddh/src/app/permissions/components/details/permission.details.html

```diff
@@ -33,20 +33,25 @@
             <div class="row mr-b2">
                 <div class="col-sm-12 form-inline">
                     <div class="form-group custom-select position-relative bg-transparent mr-r15">
-                        <dropdown-field
+                        
+                        <reactive-dropdown-field
+                            [label]="localeData?.option_select_pages"
                             [placeholder]="localeData?.option_select_pages"
-                            [readonly]="newRole?.isFixed"
-                            [required]="true"
-                            [name]="'pageName'"
+                            [enableDynamicSearch]="true"
+                            [isPaginationEnabled]="true"
```

Comment:
Check these value are available in "reactive-dropdown-field" or not 

Make changes accordingly